building/build_options.adoc

```diff
@@ -0,0 +1,67 @@
+# Options for building JBoss Tools
+
+## Maven options
+
+Here are the options we use for JBoss Tools builds:
+
+* `-e` or `--errors` if the build should fail, produce execution error messages (stack trace) to help determine what happened.
+
+* `-fae` or `--fail-at-end` continues the build until the end even when an error is detected. It allows for having more feedback at once: you get feedback for your whole build instead of only the parts that were built before the failure.
+
+* `-fn` or `--fail-never` continues the build and reports errors, but does not return a failure code. This is useful if you want your build to successfully log errors, rather than failing if errors occur. 
+
+* `-q` or `--quiet` produces quiet output - only shows errors.
+
+* `-X` or `--debug` produces execution debug output; you should DEFINITELY log to a file if using this option.
+
+* `-l` or `--log-file some-log-file.txt` defines the log file into which all build output will go. Can also pipe output through `tee` if you want to see output in console AND save to a file, eg., `mvn verify -X | tee some-log-file.txt`
+
+* `-o` or `--offline` will work offline.
+
+* `-U` or `--update-snapshots` forces a check for updated releases and snapshots on remote repositories.
+
+For the complete list of options, see http://www.sonatype.com/books/mvnref-book/reference/running-sect-options.html[Maven Command Line Options].
+
+## JBoss Tools profiles
+
+Most user-oriented JBoss Tools profiles are meant to give you control on the dependency resolution.
```

Comment:
I think it means, "The profiles listed below give you control of the dependency resolution by defining which target platform to use." So I've changed it to that.